pylabrobot/resources/plate.py

```diff
@@ -44,7 +46,7 @@ def serialize(self) -> dict:
     return {**super().serialize(), "nesting_z_height": self.nesting_z_height}
 
 
-class Plate(ItemizedResource[Well]):
+class Plate(ResourceHolderMixin, ItemizedResource[Well]):
```

Comment:
why does plate need this? i thought this class was specifically for resources that hold plates